recipe/meta.yaml

```diff
@@ -10,15 +10,17 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  skip: true  # [osx]
 
 requirements:
   run:
     - cuda-cupti-dev 12.0.90
-    - cuda-gdb 12.0.90
+    # cuda-gdb should be supported on ppc64le, but is currently broken
+    # ref https://github.com/conda-forge/cuda-gdb-feedstock/pull/4
+    - cuda-gdb 12.0.90       # [linux64 or aarch64]
     - cuda-nvdisasm 12.0.76
-    - cuda-nvprof 12.0.90         # [not aarch64]
-    - cuda-nvtx 12.0.76
+    - cuda-nvprof 12.0.90    # [not aarch64]
+    - cuda-nvtx 12.0.76      # [linux]
```

Comment:
Should we use `cuda-nvtx-dev` on Windows or does that not make sense (given it would be inconsistent with other platform packages)?

Reply:
Let's leave it consistent for now, can always add it down the road if we find a use for it.